examples/06-add_load/main.cpp

```diff
@@ -0,0 +1,122 @@
+// 02-update_model:
```

Comment:
Is it part of your PR ? I thought this was what Adrian is working on.

Reply:
I added the function since I needed to use it for some functions in motion force task.  I believe Adrian was going to work on this feature.